### PR TITLE
Minor release. For coq-8.12/13

### PR DESCRIPTION
**Pull request by Matafou**
Updating to 8.12 and removing a warning at compile time.

### CODE REVIEW

**Comment by Matafou** [comment]
Had to make a minor release to remove a warning at compile time.

**Comment by palmskog** [comment]
@Matafou but don't you want to change the URL in the package to point to the tarball for 1.0.1? Currently, it still points to the old tarball for 1.0.0.

**Comment by Matafou** [comment]
Rhaa I thought I had done it. Done now.

File: released/packages/coq-libhyps/coq-libhyps.1.0.1/opam [hunk 1]
```diff
@@ -38,6 +38,6 @@ premisses, but define your own manipulation on (groups of) hypothesis.
 "
 
 url {
-    http: "https://github.com/Matafou/LibHyps/archive/libhyps-1.0.0.tar.gz"
+    http: "https://github.com/Matafou/LibHyps/archive/libhyps-1.0.1.tar.gz"
     checksum: "07830a474ce69f1040677141880995c2"
```
Comment (palmskog) [inline released/packages/coq-libhyps/coq-libhyps.1.0.1/opam:42]:
```suggestion
    checksum: "f15584ad68bc77a5bc06048abdd9aa0d"
```

**Comment by Matafou** [comment]
@palmskog  sorry to take your time. I have a question: In april I did this #1224 but it was not taken into account (opam still did not accept 8.11). What should I have done?
And do I need to do something now, except wait for the merge?

**Comment by palmskog** [comment]
Not sure I understand the first question, but if you want to support 8.11, you need to have `< "8.12~"`, and to support 8.12 you need `< "8.13~"`, etc. 

Nothing further needs to be done now, I think.

**Comment by Matafou** [comment]
Thanks @palmskog!
OK So for instance if I propose now a PR with "< 8.13~" into "< 8.14~" it will work for 8.13 without any other action?

**Comment by palmskog** [comment]
@Matafou the reason we allow `< 8.13~` is that it's at least possible to know if a development works with 8.12 by trying 8.12+beta1. However, right now I will personally decline a PR for the `released` repo with `< 8.14~`, since we have no idea what 8.13 will look like. If you don't want to bother with handling compatibility, you can, e.g., set `>= 8.11` and nothing else. However, this means that maintainers will simply set an upper bound on the package if we discover it breaks, and there are basically no guarantees for users whether it will build at any time with their version.

**Comment by Matafou** [comment]
I understand. It is just that when compiling current coq master I get 8.13xxx. So if people use this version number somewhere in there own opam sources it will not work.

**Pull request merged**